MAINTAINING.md

```diff
@@ -155,6 +155,28 @@ GitHub's review sidebar.
 If there are multiple maintainers reviewing a PR, always give the other
 maintainers reasonable time to ACK before dismissing their review.
 
+### Release, Feature Freeze, and Backports
+
+Before the official release of a new RIOT version, two feature freeze periods
+are announced on the
+[RIOT development email list](https://lists.riot-os.org/mailman/listinfo/devel):
+The soft feature freeze and the hard feature freeze. During the soft feature
+freeze only PRs with minor impact should be merged into master. The hard feature
+freeze begins when the release manager creates a new release branch. Therefore,
+the restriction on merging PRs into the master branch are lifted at that point.
+
+Once the release branch is created, no new features should be backported.
```

Comment:
I would now prefer to reword it to "Once the release branch is created, no backports of new features should be accepted". I think that we cannot expect e.g. a new contributor to be aware of the release process. The original sentence would however blame that person, if she/he creates a PR to backport a feature.

I think however it is fair to expect that maintainers are aware of `MAINTAINING.md` and its contents.

Reply:
I would like to rephrase to `"Once the release branch is created, no backports of new features will be accepted"`.

Reply:
@fjmolinas: Added, with slightly different wording. Can you confirm you're fine with that?